apps/desktop/src/i18n/catalog.ts

```
@@ -3,10 +3,12 @@ import { en } from './en'
 import { ja } from './ja'
 import type { Locale, Translations } from './types'
 import { zh } from './zh'
+import { ptBr } from './pt-br'
```

ESLint

[error] Expected "./pt-br" to come before "./zh".

(perfectionist/sort-imports)